lib/rake/rspec.rb

```diff
@@ -0,0 +1,173 @@
+# encoding: utf-8
+require_relative "../bootstrap/environment"
+require 'pathname'
+require_relative "../pluginmanager/gemfile"
+
+class GemfileHelpers
```

Comment:
Should this be a module? It doesn't seem very object like. At the very least maybe a singleton?